core/templates/dev/head/domain/exploration/StatesObjectFactory.js

```diff
@@ -107,6 +107,18 @@ oppia.factory('StatesObjectFactory', [
       return finalStateNames;
     };
 
+    States.prototype.getAllAudioLanguageCodes = function() {
+      var allAudioLanguageCodes = new Set();
```

Comment:
Set is relatively recent and isn't supported in some slightly older browsers. Might want to consider using an array instead and checking for existing membership with indexOf().